### PR TITLE
Update serving runtime version for 0.12 release and add some notes

### PR DESCRIPTION
**Pull request by sivanantha321**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

- Update serving runtime version for 0.12 release
- Add warning about deploying inference services in control plane namespaces.
- Update reference docs for 0.12 release.
- Add a note on quick install failure.

### CODE REVIEW

File: docs/get_started/README.md [hunk 1]
```diff
@@ -41,3 +40,9 @@ The [Kubernetes CLI (`kubectl`)](https://kubernetes.io/docs/tasks/tools/install-
     curl -s "https://raw.githubusercontent.com/kserve/kserve/release-0.11/hack/quick_install.sh" | bash
     ```
 
+!!! Note
+    If you encounter the following error when installing, Please re-run the script.
```
Comment (yuzisun) [inline docs/get_started/README.md:44]: How often do we see this issue ?

File: docs/modelserving/v1beta1/serving_runtime.md [hunk 2]
```diff
@@ -37,16 +37,16 @@ In a given serving runtime directory the pyproject.toml file contains the exact
...
-| [XGBoost ModelServer](https://github.com/kserve/kserve/tree/master/python/xgbserver)            | [Saved Model](https://xgboost.readthedocs.io/en/latest/tutorials/saving_model.html)                                           | v1, v2              | v2          | v0.11 (KServe)                                                               | 1                                                                                                                                                         | [XGBoost Iris](./xgboost/README.md)               |
+| [XGBoost ModelServer](https://github.com/kserve/kserve/tree/master/python/xgbserver)            | [Saved Model](https://xgboost.readthedocs.io/en/latest/tutorials/saving_model.html)                                           | v1, v2              | v2          | v0.12 (KServe)                                                               | 1                                                                                                                                                         | [XGBoost Iris](./xgboost/README.md)               |
```
Comment (yuzisun) [inline docs/modelserving/v1beta1/serving_runtime.md:49]: Can we also add huggingface runtime ?

File: docs/get_started/README.md [hunk 3]
```diff
@@ -41,3 +40,9 @@ The [Kubernetes CLI (`kubectl`)](https://kubernetes.io/docs/tasks/tools/install-
     curl -s "https://raw.githubusercontent.com/kserve/kserve/release-0.11/hack/quick_install.sh" | bash
```
Comment (yuzisun) [inline docs/get_started/README.md:40]:
```suggestion
    curl -s "https://raw.githubusercontent.com/kserve/kserve/release-0.12/hack/quick_install.sh" | bash
```

File: docs/get_started/README.md (hunk 1 above)
Comment (sivanantha321) [reply docs/get_started/README.md:44]: Removing this for now

File: docs/get_started/README.md (hunk 3 above)
Comment (sivanantha321) [reply docs/get_started/README.md]: Done

**Pull request merged**